docs/deployment/release.md

```diff
@@ -25,77 +27,42 @@ version numbers look like: `YYYY.0M.R`
 
-Typically changes for a release will move from `main`, to `test`, to `prod`. This
-assumes `main` is in a state that it can be deployed without disruption. (This is called a `Regular` release.)
+## 1. Create a release candidate tag on `main` and push it
 
-If `main` or `test` contain in-progress work that is not ready for production,
-and a hotfix is needed in production, a separate process to test the changes
-before deploying to `prod` must be undertaken. (This is called a `Hotfix` release.)
-
-As implied in the previous step, all releases follow the same version number format.
-
-The following diagram shows how a release should propagate to `prod` under
-different circumstances:
-
-```mermaid
-graph LR
-    A(Release branch) --> B{Are main and test ready to deploy?};
-    B -->|Yes| C(main);
-    C --> D(test);
-    D --> E(prod);
-    B -->|No| E;
+```bash
+git fetch
+git checkout main
+git reset --hard origin/main
+git tag -a YYYY.0M.R-rcR
 ```
 
-By convention the release branch is called `release/YYYY.0M.R` using the
-upcoming version number.
-
-## 2. Bump the application version number
-
-The app code maintains a version number in
-[`benefits/__init__.py`](https://github.com/cal-itp/benefits/blob/main/benefits/__init__.py),
-used by the instrumentation and logging systems.
-
-This version number must be updated to match the new version in the same format:
-`YYYY.0M.R`
-
-## 3. Open a PR
-
-Initially from the release branch to the target environment branch, following
-the merge sequence in the diagram above.
+Git will open your default text editor and prompt you for the tag annotation. For the tag annotation, use the release candidate version. Finally, after closing the text editor:
 
-## 4. Merge the PR
-
-After checks pass and review approval is given, merge the PR to kick off the
-deployment.
-
-Repeat steps 3 and 4 for each deployment environment target, again following the
-merge sequence in the diagram above.
-
-## 5. Tag the release
+```bash
+git push origin YYYY.0M.R-rcR
+```
 
-Once the deploy has completed to `prod`, the version can be tagged and pushed to
-GitHub.
+This builds a new package and deploys to the Azure test environments. No GitHub release is created for release candidates.
 
-From a local terminal:
+## 2. Create a release tag on `main` and push it
 
 ```bash
 git fetch
-
-git checkout prod
-
-git reset --hard origin/prod
-
+git checkout main
+git reset --hard origin/main
 git tag -a YYYY.0M.R
 ```
 
-Git will open your default text editor and prompt you for the tag annotation. For the tag annotation, use the title of the `release`-tagged Issue that kicked off the release. Finally, after closing the text editor:
+Git will open your default text editor and prompt you for the tag annotation. For the tag annotation, use the title of the release-tagged Issue that kicked off the release. Finally, after closing the text editor:
```

Comment:
We've typically been calling these types of issues _Release process issues_

Reply:
I think this wording is mirroring eligibility-server's documentation, which means I am to [blame](https://github.com/cal-itp/eligibility-server/blame/main/docs/releases.md#L63) 😅 \*rimshot\*